docs/setup/install/installation-configuration.md

```diff
@@ -36,4 +36,19 @@ helm install devtron devtron/devtron-operator --create-namespace --namespace dev
 |INGRESS_ANNOTATIONS | Annotations for ingress| ""| Optional|
 |PROMETHEUS_URL | Existing Prometheus URL if it is installed| ""| Optional|
 
+### Dashboard Configurations
+
+```bash
+RECOMMEND_SECURITY_SCANNING=false
+FORCE_SECURITY_SCANNING=false
+HIDE_DISCORD=false
+```
+
+|Parameter | Description|
+|-|-|
+|RECOMMEND_SECURITY_SCANNING | By default `security scanning` is `enabled` for a new build pipeline. User can turn it off.|
```

Comment:
If True `security scanning` is `enabled` by default for a new build pipeline. Users can however turn it off in the new or existing pipelines.